<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>474</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>474</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EyeNet - Filter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/logo.png</normaloff>../resources/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* { border:none }‚Äè

QToolTip
 {
     border: 0.1ex solid #eff0f1;
     background-color: #31363b;
     alternate-background-color: #3b4045;
     color: #eff0f1;
     padding: 0.5ex;
     opacity: 200;
 }
 
 QWidget
 {
     color: #eff0f1;
     background-color: #31363b;
     selection-background-color:#3daee9;
     selection-color: #eff0f1;
     background-clip: border;
     border-image: none;
     border: 0px transparent black;
     outline: 0;
 }
 
 QWidget:item:hover
 {
     background-color: #3daee9;
     color: #eff0f1;
 }
 
 QWidget:item:selected
 {
     background-color: #3daee9;
 }
 
QLineEdit
 {
     background-color: #232629;
     padding: 0.5ex;
     border-radius: 0.2ex;
     color: #FAFAFA;
 }
QLineEdit:hover
 {
     border: none;  /* we wont change this i guess */
     color: #eff0f1;
 }
 QLineEdit:hover:pressed
 {
     background-color: #31363b; 
 }

 
 QGroupBox
 {
     border:1px solid #76797c;
     border-radius: 5px;
     padding-top: 1ex;
     margin-top: 1ex;
 }
 
 QGroupBox::title
 {
     subcontrol-origin: margin;
     padding-left: 0.1ex;
     padding-right: 0.1ex;
     /*margin-top: -10px;*/
 }

QPushButton
 {
     color: #eff0f1;
     background-color: qlineargradient(x1: 0.5, y1: 0.5 x2: 0.5, y2: 1, stop: 0 #3b4045, stop: 0.5 #31363b);
     border-color: #76797c;
     border-style: solid;
     padding: 0.5ex;
     border-radius: 0.2ex;
     outline: none;
 }
 
 QPushButton:disabled
 {
     background-color: #31363b;
     border-width: 0.1ex;
     border-color: #454545;
     border-style: solid;
     padding-top: 0.5ex;
     padding-bottom: 0.5ex;
     padding-left: 1ex;
     padding-right: 1ex;
     border-radius: 0.2ex;
     color: #454545;
 }
 
 QPushButton:focus
 {
     color: white;
 }
 
 QPushButton:pressed
 {
     background-color: #31363b;
     padding-top: -1.5ex;
     padding-bottom: -1.7ex;
 }
 
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Basic Filters</string>
   </property>
   <widget class="QLineEdit" name="macsrc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>MAC src:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="macdst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>MAC dst:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipsrc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>IP src:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipdst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>IP dst:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="datecap">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date (From):</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="datecap2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date (To):</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Network Filters</string>
   </property>
   <widget class="QLineEdit" name="netip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Network ip:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="netmask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Netmask:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="icmpfilter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>ICMP Filters</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="icmptype">
       <property name="placeholderText" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="tcpfilter">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>TCP Filters</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="tcpportsrc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Port Src:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpportdst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Port Dst:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uflag">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="aflag">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pflag">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sflag">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fflag">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rflag">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>98</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Flags:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="udpfilter">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>UDP Filters</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="udpportsrc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Port Src:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="udpportdst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Port Dst:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ipv6filter">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>IPv6 Filters</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="ipv6src">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>IPv6 Src:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipv6dst">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>IPv6 Dst:</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="arpfilter">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ARP</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
